### DISCUSSION TITLE
zendesk-java-client 0.18.0 release is out

### DISCUSSION THREAD

**Discussion by PierreBtz** [cloudbees/zendesk-java-client#492] [Announcements]
Here is a new release with few new end-points and a batch of dependencies updates.

*Thanks a lot to our contributor @rcprcp*

## 🚀 New features and improvements

* Add basic View support - List the Views; List the tickets in a View (#491) @rcprcp

## 📦 Dependency updates

* Bump netty-bom from 4.1.77.Final to 4.1.78.Final (#490) @dependabot
* Bump maven-enforcer-plugin from 3.0.0 to 3.1.0 (#488) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/cloudbees-oss/zendesk-java-client/releases/tag/zendesk-java-client-0.18.0'>0.18.0</a>.</em>

**Comment by evlyngeorge** [comment]
Hi @PierreBtz Can you please include this api in incremental export. This include Parameter is not accepted by the existing api's. We need this for our project.
https://{subdomain}.zendesk.com/api/v2/incremental/tickets.json?start_time=1332034771&include=metric_sets
